router/options.go

```diff
@@ -0,0 +1,105 @@
+package router
+
+import (
+	"github.com/google/uuid"
+	"github.com/micro/go-micro/registry"
+)
+
+var (
+	// DefaultGossipAddress is default gossip bind address
+	DefaultGossipAddress = ":9093"
+	// DefaultNetworkAddress is default network bind address
+	DefaultNetworkAddress = ":9094"
+)
+
+// Options are router options
+type Options struct {
+	// ID is router id
+	ID string
+	// Address is router micro service address
+	Address string
+	// GossipAddress is router gossip address
+	GossipAddress string
```

Comment:
Change gossip address to Address as no one knows its a gossip registry and the assumption is just this is the router's address to talk to other routers. We just happen to use gossip for it.